docs/release-notes/highlights-0.9.0.asciidoc

```diff
@@ -29,4 +29,7 @@ this allows you to configure your own init containers to run code before Elastic
 Or you can use the Pod template to mount additional volumes with configuration files, for example to configure SAML or synonym files,
 or to specify `imagePullSecrets` for your private Docker registry.
 
+[float]
+=== Breaking Changes
 
+Existing resources (Elasticsearch and Kibana) that were created with a 0.8.x version of the operator are *not* compatible with this release. They will automatically have an annotation added to them indicating the controller version, and then skipped. They will continue to function, but any changes made to them will not be processed by the 0.9.x version of the operator. To upgrade existing clusters, please https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html[perform a snapshot] of the existing cluster and restore them into a new cluster created by the 0.9.x version of the operator.
```

Comment:
Reindex from remote is also an option https://www.elastic.co/guide/en/elasticsearch/reference/current/reindex-upgrade-remote.html

Reply:
Ah good point